iModelJsNodeAddon/JsInteropDgnDb.cpp

```diff
@@ -871,7 +871,7 @@ Napi::String JsInterop::InsertLinkTableRelationship(DgnDbR dgndb, Napi::Object o
     BeSQLite::EC::ECInstanceKey relKey;
     auto rc = dgndb.InsertLinkTableRelationship(relKey, *relClass, sourceId, targetId, props.get()); // nullptr is okay if there are no props
     if (BE_SQLITE_OK != rc)
-        throwSqlError();
+        throwSqlError(rc);

```

Comment:
Could be something like following, may be with source/target id.

```cpp
BeNapi::ThrowJsException(
    obj.Env(), 
    SqlPrintfString("Failed to insert relationship (%s)", relClass->GetFullName()).GetUtf8CP(), 
    (int) rc
);
```

Reply:
Proposing to instead use:
```
       JsInterop::throwSqlResult(
          "Failed to insert relationship",
          dgndb.GetDbFileName(),
          rc
        );
```
I also modified throwSqlResult to return the entire log error string, so it would look like this:
Error: Failed to insert relationship [D:\itwinjs-core\core\backend\lib\cjs\test\output\IModel\insertDuplicateInstance.bim]: rc=2067, constraint failed (BE_SQLITE_CONSTRAINT_UNIQUE)